contract-rs/01-basic-auction/Cargo.toml

```diff
@@ -9,10 +9,10 @@ crate-type = ["cdylib", "rlib"]
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
-near-sdk = { version = "5.3" }
+near-sdk = { version = "5.4.0" }
```

Comment:
The latest near-sdk version is 5.5.0. Could you try to update it in dependencies?